spec/acceptance/default_parameters_spec.rb

```diff
@@ -40,17 +40,17 @@ class { 'jira':
     EOS
     pp_upgrade = <<-EOS
       $java_package = $facts['os']['family'] ? {
-        'RedHat' => 'java-11-openjdk-headless',
-        'Debian' => 'openjdk-11-jre-headless',
+        'RedHat' => 'java-17-openjdk-headless',
+        'Debian' => 'openjdk-17-jdk-headless',
       }
 
       $java_home = $facts['os']['family'] ? {
-        'RedHat' => '/usr/lib/jvm/jre-11-openjdk',
-        'Debian' => '/usr/lib/jvm/java-1.11.0-openjdk-amd64',
+        'RedHat' => '/usr/lib/jvm/jre-17-openjdk',
+        'Debian' => '/usr/lib/jvm/java-17-openjdk-amd64',
       }
 
       class { 'jira':
-        version                  => '8.16.0',
+        version                  => '9.12.18',
```

Comment:
The version parameter in init.pp still defaults to 8. Should that be updated as well? Or do we want to keep the old version and test if the module can upgrade jira 8 to 9?

Reply:
Jira 8.x, which has been eol a year now, does not support Java 17.
I guess most module users have moved to 9 by now, so I vote for bumping the default version :)

Reply:
Hopefully that will solve the Debian 12/Ubuntu 24.04 issues as well.